### DISCUSSION TITLE
The Artery chip

### DISCUSSION THREAD

**Discussion by Felixleo-zx** [keirf/flashfloppy#455] [General]
Gotek installs Other Artery Chip (AT32F415CBT7 Cortex-M4) !
Can you modify the firmware for this chip?
![2021-04-08 04-33-16](https://user-images.githubusercontent.com/82413201/114455506-81f38100-9be4-11eb-8aa4-2813c383cf2d.JPG)

**Comment by keirf** [comment]
Duplicates #454

**Reply by Felixleo-zx** [reply]
Thanks for the tip.

**Comment by keirf** [comment]
Ah this is a discussion thread not a ticket. Then the answer is yes: see the ticket referenced above.

**Reply by Felixleo-zx** [reply]
I really hope that you firmware will be available, thank you!
